x-pack/plugins/observability_solution/observability_ai_assistant_app/server/functions/visualize_esql.ts

```diff
@@ -36,16 +58,16 @@ export function registerVisualizeESQLFunction({
           name,
           meta: { type: esFieldTypeToKibanaFieldType(type) },
         })) ?? [];
+
+      const message = getSystemMessage(intention, query, errorMessages);
+
       return {
         data: {
           columns,
+          errorMessages,
```

Comment:
ℹ️ I could also send here only the first error message reported (I don't expect that the assistant will fail in 2 places tbh). I could also send the first x characters of the message. It is usually short.

Reply:
I think sending them all is fine. But, to get them to the LLM they have to be in `content`, not `data`. `data` can only be accessed by render functions, content is what gets sent to the LLM as well.

Reply:
I have also added them in the `content` like that

```
'The query has syntax errors: ```\n' + queryErrors + '\n```';
```